src/UserAgreement/Quiz.less

```diff
@@ -0,0 +1,84 @@
+.quiz__form {
+  display: block;
+  width: 1025px;
+  background-color: var(--g3-color__white);
+}
+
+.quiz__questions {
+  display: inline-grid;
+  overflow-y: scroll;
+  overflow-x: hidden;
+  float: left;
+  width: 740px;
+  height: 740px;
+  background-color: var(--g3-color__white);
+}
+
+.quiz__title {
+  width: 100%;
+  height: 70px;
+  line-height: 70px;
+  padding: 0 30px;
+  vertical-align: middle;
+  background: var(--g3-color__base-blue);
+  color: #ffffff;
+  display: block;
+}
+
+.quiz__icon {
+  float: left;
+  width: 50%;
+  display: inline-block;
+}
+
+.quiz__title-text {
+  display: inline-block;
+}
+
+.quiz__menu {
+  display: inline-block;
+  float: right;
+  text-align: right;
+  width: 50%;
+}
+
+.quiz__menu-bullet {
```

Comment:
The text here is not precisely centered aligned, how about following changes: 
1. Change `.quiz__menu-bullet`'s display style to  `display: inline-block;`, remove its all paddings
2. For `.quiz__menu-bullet.h3`, add `text-align: center;` 
It will fix this :)